sdk/storage/Azure.Storage.Blobs/swagger/readme.md

```diff
@@ -147,6 +147,16 @@ directive:
     }
 ```
 
+### Make CORS allow null values
+It should be possible to pass null for CORS to update service properties without changing existing rules.
+``` yaml
+directive:
+- from: swagger-document
+  where: $.definitions.BlobServiceProperties
```

Comment:
We need to do this in the swagger readme.md's for Queues and Files as well.  They both get renamed to `QueueServiceProperties` and `FilesServiceProperties`.

Reply:
Ohh, I just realized I had made those changes in my local but forgot to push :(